tests/worker/test_workflow.py

```diff
@@ -222,7 +220,7 @@ async def test_workflow_info(client: Client, env: WorkflowEnvironment):
         assert info["retry_policy"] == json.loads(
             json.dumps(dataclasses.asdict(retry_policy), default=str)
         )
-        assert uuid.UUID(info["run_id"]).version == 4
+        assert uuid.UUID(info["run_id"]).version == 7
```

Comment:
The server switched to v7 UUIDs for runID at https://github.com/temporalio/temporal/pull/6933